airflow/www/static/js/dag/details/taskInstance/Logs/utils.ts

```diff
@@ -62,6 +63,16 @@ export const parseLogs = (
   const fileSources: Set<string> = new Set();
   const ansiUp = new AnsiUp();
 
+  const urlRegex = /((https?:\/\/|http:\/\/)[^\s]+)/g;
+  // Detect log groups which can be collapsed
+  // Either in Github like format '::group::<group name>' to '::endgroup::'
+  // see https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines
+  // Or in ADO pipeline like format '##[group]<group name>' to '##[endgroup]'
+  // see https://learn.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=powershell#formatting-commands
+  const logGroupStart = / INFO - (::|##\[])group(::|\])([^\n])*/g;
+  const logGroupEnd = / INFO - (::|##\[])endgroup(::|\])/g;
+  const logGroupStyle = "color:#0060df;cursor:pointer;font-weight: bold;";
```

Comment:
Could you leave a comment on what color this hex is?

Reply:
Actually I did not want to hard-code a color here - I just selected the same color like we have with other areas of "blue". Added a comment - if there is a proper/better was to use CSS let me know. Could not find any reasonable way but would like to also offer making it better.